<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QDialog" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-uncertain">
    <normaloff>clueless.png</normaloff>clueless.png</iconset>
  </property>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUsername">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Registration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>RegisterForm</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
